src/marks/geo.js

```diff
@@ -95,3 +94,19 @@ export function sphere({strokeWidth = 1.5, ...options} = {}) {
 export function graticule({strokeOpacity = 0.1, ...options} = {}) {
   return geo(geoGraticule10(), {strokeOpacity, ...options});
 }
+
+export function maybeGeometry(geometry) {
+  const {value, scale} = geometry?.value ? geometry : {value: geometry};
```

Comment:
I expect we want to use maybeValue here for consistency.

Reply:
More importantly, it should be necessary to apply the scale override here. Scale overrides are applied here:

https://github.com/observablehq/plot/blob/26b5d14ca94697efb1229c38b2ebbc3fb4318291/src/mark.js#L48-L52

We probably need to add some logic to inferChannelScale to handle the _projection_ scale being applied to the _geometry_ channel.